js/views/modals/purchase/Payment.js

```diff
@@ -142,6 +147,7 @@ export default class extends BaseVw {
           this.showSpendError(err);
           recordEvent('Purchase_SpendFromWallet', {
             currency,
+            page: this.page,
             errors: err || 'unknown error',
```

Comment:
Can we be consistent with `none` vs. `unknown error`?

Reply:
It is. This is in the fail handler, so it's not 'none', 'none' means there were no errors.

I noticed sometimes err is undefined, I'd rather record that as 'unknown error' in the metrics so it's more clear to Jenn and Sam.